go.mod

```diff
@@ -59,21 +59,21 @@ require (
 	go.etcd.io/etcd/tests/v3 v3.5.11
 	go.nhat.io/aferocopy/v2 v2.0.1
 	go.opencensus.io v0.24.0
-	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.46.0
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.48.0
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.44.0
```

Comment:
Updated lib, to fix data race, but may be we have to wait for `0.48.1`:
https://github.com/open-telemetry/opentelemetry-go-contrib/issues/4895